src/main/java/uk/gov/di/ipv/cri/common/library/service/ConfigurationService.java

```diff
@@ -128,4 +151,9 @@ public String getKmsEncryptionKeyId() {
     private String getParameterName(SSMParameterName parameterName) {
         return String.format(PARAMETER_NAME_FORMAT, parameterPrefix, parameterName.parameterName);
     }
+
+    private static String getEnvironmentVariableValue(String envVar) {
+        return Objects.requireNonNull(
```

Comment:
This method can be removed because the `AWS_STACK_NAME` environment variable will always be present (as per the `Globals` in template.yaml )